### ISSUE TITLE
mkdocs serve: Cairo causes failure on windows

### ISSUE THREAD

**Issue by agardnerIT** [open-feature/flagd#862]
## TLDR: Comment out the social plugin

In `mkdocs.yml`, just comment out the `plugins` and `- social` lines.

Then run `mkdocs serve` again.

<img width="412" alt="image" src="https://github.com/open-feature/flagd/assets/26523841/9df83cf3-4098-4b6a-a99e-07c083fda129">

## Description of Pain

Running `mkdocs serve` while authoring docs is painful on Windows because Cairo is not natively available on windows.

```
OSError: no library called "cairo-2" was found
no library called "cairo" was found
no library called "libcairo-2" was found
cannot load library 'libcairo.so.2': error 0x7e.  Additionally, ctypes.util.find_library() did not manage to locate a library called 'libcairo.so.2'
cannot load library 'libcairo.2.dylib': error 0x7e.  Additionally, ctypes.util.find_library() did not manage to locate a library called 'libcairo.2.dylib'
cannot load library 'libcairo-2.dll': error 0x7e.  Additionally, ctypes.util.find_library() did not manage to locate a library called 'libcairo-2.dll'
```

`cairo` is required by the social plugin.

I realise that [this page](https://squidfunk.github.io/mkdocs-material/setup/setting-up-social-cards/) lists prerequisites and dependencies, pointing to [this page](https://squidfunk.github.io/mkdocs-material/setup/dependencies/image-processing/) and does mention that Windows requires GTK+.

All of that said, I still find it cumbersome that I need to clutter up my system with GTK, just to get a social plugin working.

**Comment by beeme1mr** [comment]
I had this problem too. Perhaps you could try running it in a container.

https://squidfunk.github.io/mkdocs-material/creating-your-site/#previewing-as-you-write
